### PR TITLE
Run an installation task to install a language

### PR DESCRIPTION
**Pull request by poncovka**
The lang command will run a remote installation task to install
a language on the installed system.

### CODE REVIEW

File: pyanaconda/modules/localization/installation.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/modules/localization/installation.py]: I am not sure about this. I would prefer to sort tasks by their functionality. For example in the storage module, you don't want to have all tasks in `pyanaconda.modules.storage.tasks`, but you might want to keep them close to other related files, for example the bootloader tasks should be in `pyanaconda.modules.storage.bootloader`.

File: pyanaconda/modules/localization/installation.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        super().__init__()
+        self._sysroot = sysroot
+        self._lang = lang
+
+    @property
+    def name(self):
+        return "Configure language"
+
+    def run(self):
+        write_language_configuration(self._lang, self._sysroot)
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/modules/localization/installation.py]: Ok, I can move it there.

File: pyanaconda/modules/localization/installation.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkonecny12) [reply pyanaconda/modules/localization/installation.py]: And what about sorting then inside of that subfolder?

[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/modules/localization/installation.py]: I don't know. What do @rvykydal and @M4rtinK think?

**Comment by jkonecny12** [comment]
jenkins, test this please

File: pyanaconda/modules/localization/installation.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/modules/localization/installation.py]: Fixed.

File: pyanaconda/modules/localization/installation.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/modules/localization/installation.py]: We agreed that tasks will not be in a subfolder `tasks`.

**Comment by poncovka** [comment]
**Updated:** Created a specific exception and added new tests for the module and the task.

**Pull request merged**